Assets/Scripts/Controllers/UIController.cs

```diff
@@ -1,23 +0,0 @@
-﻿#region License
```

Comment:
My thought is, let's add it back when it's actually being used, but for right now, I'm not convinced it's needed.